commands/team_test.go

```diff
@@ -108,3 +108,207 @@ func (s *MmctlUnitTestSuite) TestCreateTeamCmd() {
 		s.Require().Len(printer.GetLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveUserCmd() {
+	teamArg := "example-team-id"
+	userArg := "example-user-id"
+	s.Run("Remove users from team, no args", func() {
```

Comment:
I would suggest to give the test names as proper sentences, not short snippets. For example, for this one, I would suggest: "Remove users from team without args returns an error". This comment applies to all other tests too.